dll/win32/imm32/imm.c

```diff
@@ -3129,25 +3129,29 @@ BOOL WINAPI ImmSetOpenStatus(HIMC hIMC, BOOL fOpen)
  */
 BOOL WINAPI ImmSetStatusWindowPos(HIMC hIMC, LPPOINT lpptPos)
 {
-    InputContextData *data = get_imc_data(hIMC);
+    LPINPUTCONTEXT pIC;
+    HWND hwnd;
+    DWORD idImeThread, idThread;
 
-    TRACE("(%p, %p)\n", hIMC, lpptPos);
+    TRACE("ImmSetStatusWindowPos(%p, {%ld, %ld})\n", hIMC, lpptPos->x, lpptPos->y);
 
-    if (!data || !lpptPos)
-    {
-        SetLastError(ERROR_INVALID_HANDLE);
+    idImeThread = Imm32QueryInputContext(hIMC, 1);
```

Comment:
Do you know what this "1" is? Could it be the same as the one I've commented about in your other PR?

Reply:
I don't know at all.

Reply:
Hypothesis: `Imm32QueryInputContext`'s 2nd paramter is same as `Imm32QueryWindow`'s 2nd parameter.